src/parser/github-comment-module.ts

```diff
@@ -58,6 +69,22 @@ export class GithubCommentModule implements Module {
     );
     bodyArray.push("\n-->");
```

Comment:
I'm confused what's going on here? It looks like you're closing an html comment, but I don't see the opening of it.

Reply:
https://github.com/ubiquibot/conversation-rewards/blob/a9f3a2958babf72362abd56b0fb82b0d571ead77/src/parser/github-comment-module.ts#L66